File: Aula_3/circulo.py
import math 
class circulo:
    def __init__(self):
         self.r = 0
    def area(self):
        return math.pi * (self.r ** 2)
    def circumferencia(self):
        return 2 * math.pi * self.r

x = circulo()
x.r = 5
x.r = x.r/2
print(x.r)
print(x.area())
print(x.circumferencia())

File: Aula_3/circulo.cpp
#include <iostream>
#include <cmath>
using namespace std;

class circulo{
public:
    double raio;
    double area() {
        return M_PI * raio * raio;
    }
    double circumferencia() {
        return M_PI * raio * 2;
    }
};

int main() {
 circulo x;
 circulo y;
 cin >> x.raio >> y.raio;
 
 cout << x.raio << " " << x.area() << " " << x.circumferencia() << endl;
 cout << y.raio << " " << y.area() << " " << y.circumferencia() << endl;
}